### PR TITLE
add company - RemoteMore

### PR DESCRIPTION
**Pull request by ridge-kimani**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes, etc.) - **If this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned above and confirm all included details are correct.
* [ ] You know your alphabet - the company is listed in alphabetical order in the README.md file.
* [ ] The company directly hires employees. ** No boot camps / freelance sites / etc**
* [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [ ] A [company profile](/company-profiles/remotemore.md) is included - This is __required__ for new additions.
* [ ] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [ ] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [ ] __How to apply__ details the best approach for new applications, a page on the site where open positions are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/remotemore.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# RemoteMore
+
+## Company blurb
+Our vision of the future is to ensure there's equal access to job opportunities. It is highly embodied
```
Comment (KieranRobson) [inline company-profiles/remotemore.md:4]: Please provide further detail regarding what the company actually does.

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change.

**Review by KieranRobson — APPROVED** [review]
LGTM - Thanks!

**Comment by dougaitken** [comment]
@ridge-kimani please review the Pull Request checklist which you have ignored. 

>  The company directly hires employees. ** No boot camps / freelance sites / etc**

This company hires directly and hires freelancers - please ensure your addition only focuses on direct hires, i.e. the `https://remotemore.com/careers`  section

**Review by dougaitken — APPROVED** [review]

**Pull request merged**